include/rmm/detail/aligned.hpp

```diff
@@ -71,6 +71,20 @@ constexpr std::size_t align_down(std::size_t v, std::size_t align_bytes) noexcep
   return v & ~(align_bytes - 1);
 }
 
+/**
+ * @brief Checks whether a value is aligned to a multiple of a specified power of 2
+ *
+ * @param[in] v value to check for alignment
+ * @param[in] alignment amount, in bytes, must be a power of 2
+ *
+ * @return true if aligned
+ */
+constexpr bool is_aligned(std::size_t v, std::size_t align_bytes) noexcept
```

Comment:
`mr_test.hpp` has an `is_aligned`. Should they be consolidated? (currently the implementations differ)

Reply:
Good catch. The test one tests pointers, so I renamed it to `is_pointer_aligned()`.